spec/features/user/log_mood_spec.rb

```diff
@@ -65,4 +65,45 @@
     expect(mood_entry.mood_id).to eq @mood1.id
     expect(mood_entry.entry).to eq "I'm feeling sad actually"
   end
+
+  it 'I can see a modal with my diary log entry by clicking an emoji on the calendar' do
+    stub_omniauth
+
+    visit '/'
+
+    click_on 'signing in'
+
+    expect(current_path).to eq('/dashboard')
+
+    fill_in :entry, with: "I'm feeling okay"
+    choose(id: "mood_id_#{@mood2.id}")
+    click_on 'Submit'
+
+    current_date = Date.today.day
+    find("[data-target='#modal#{current_date}']").click
+
+    expect(page).to have_button('Close')
+    expect(page).to have_content("Your Diary Entry on #{Date.today.strftime("%B %d, %Y")}")
+    expect(page).to have_content("I'm feeling okay")
+  end
+
```

Comment:
Maybe it would be a good idea to hard code an older date and make sure the date in the modal renders correctly as another assertion.